backend/src/modules/product/controllers/slider.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Controller, Get, Query } from '@nestjs/common';
+
+import { ProductService } from '../product.service';
+import { ProductResponse } from '../product.response';
+
+@Controller('api/slider-products')
+export class SliderController {
```

Comment:
Maybe it should be part of product controller because it gives products too but only for showing on slider